README.md

```diff
@@ -2,4 +2,4 @@
 
 **Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it..
```

Comment:
I think it would be better to make changes and add valuable information about your project in this file. you can follow this example https://github.com/microverseinc/readme-template